buildSrc/src/main/groovy/openhouse.iceberg-conventions-1.2.gradle

```diff
@@ -0,0 +1,11 @@
+ext {
+  icebergVersion = '1.2.0'
+}
+
+dependencies {
+  implementation('org.apache.iceberg:iceberg-bundled-guava:' + icebergVersion + "!!")
```

Comment:
Curious what !! do?

Reply:
It forcefully pins the version when there's a conflict.